specs/protocol/isthmus/configurability.md

```diff
@@ -39,9 +48,70 @@ The following actions should happen during the initialization of the `SystemConf
 - `emit ConfigUpdate.GAS_LIMIT`
 - `emit ConfigUpdate.UNSAFE_BLOCK_SIGNER`
 - `emit ConfigUpdate.EIP_1559_PARAMS`
+- `emit ConfigUpdate.OPERATOR_FEE_PARAMS`
 
 These actions MAY only be triggered if there is a diff to the value.
 
+`ConfigUpdate.OPERATOR_FEE_PARAMS` MAY be emitted. If it is not emitted, the `operatorFeeScalar` and
+`operatorFeeConstant` are set to 0 by default.
+
+### Modifying Operator Fee Parameters
+
+A new `SystemConfig` `UpdateType` is introduced that enables the modification of
+the `operatorFeeScalar` and `operatorFeeConstant` by the [`OperatorFeeManager`](../configurability.md#operator-fee-manager).
+
 ### Interface
 
-TODO: operator fee
+#### Fee Vault Config
+
+For each `FeeVault`, there is a setter for its config. The arguments to the setter include
+the `RECIPIENT`, the `MIN_WITHDRAWAL_AMOUNT` and the `WithdrawalNetwork`.
+Each of these functions should be `public` and only callable by the chain governor.
+
+Each function calls `OptimismPortal.setConfig(ConfigType,bytes)` with its corresponding `ConfigType`.
+
+##### `setBaseFeeVaultConfig`
```

Comment:
This still follows the standard L2 genesis style. We should not have `setConfig` or other setters here. There should be no diff to the `FeeVault`